include/ck_tile/ops/gemm/block/block_gemm_areg_breg_creg_v1.hpp

```diff
@@ -157,46 +165,15 @@ struct BlockGemmARegBRegCRegV1
         });
     }
 
-    CK_TILE_DEVICE static constexpr auto MakeCBlockTile()
```

Comment:
this method are used by fmha fwd/bwd operators. so remove it will cause compilation errors